### PR TITLE
I moved my app to The Pi Project organization (:

### PR DESCRIPTION
**Pull request by rigobertomolina**

### CODE REVIEW

**Comment by dkhamsing** [comment]
thx for the update
it does not use firebase?

**Comment by rigobertomolina** [comment]
Not anymore, I will be creating my own backend

File: contents.json [hunk 1]
```diff
@@ -7819,19 +7819,16 @@
...
       ],
       "description": "Study, solve problems, formula library, calculators & more! Made by a student for students.",
-      "source": "https://github.com/rigobertomolina/Pi",
+      "source": "https://github.com/thepiproject/Pi-iOS",
       "itunes": "https://itunes.apple.com/app/pi-a-students-dream/id1117743302",
       "license": "mit",
       "screenshots": [
         "http://a4.mzstatic.com/us/r30/Purple60/v4/44/b7/13/44b713c0-a307-b098-f835-33f2ec471436/screen696x696.jpeg"
       ],
-      "date_added": "Aug 29 2016",
```
Comment (dkhamsing) [inline contents.json]: Can you undo these changes below

**Comment by dkhamsing** [comment]
@rigobertomolina the branch has a conflict.. can u fix this? if not I can help you with that

**Comment by rigobertomolina** [comment]
What do I need to fix?

**Comment by dkhamsing** [comment]
You would have to rebase to bring in changes from the main repo to your fork
Let me update this for you, don't worry

**Comment by rigobertomolina** [comment]
Thank you, this is a little confusing aha
